### ISSUE TITLE
Crear tutorial configuración del proyecto en Linux

### ISSUE THREAD

**Issue by adrigm** [genbetadev/Genbeta-Dev-Engine#12]

**Comment by RicardoRB** [comment]
¿Puedo hacer un tutorial con el IDE codeblocks?

**Comment by adrigm** [comment]
Claro, ¿También te animas a crear el proyecto?

**Comment by RicardoRB** [comment]
Si te refieres a realizar un proyecto en codeblocks con SFML 2 y Genbeta-Dev-Engine, no habría problema, ahora mismo estoy usando SFML 2.1 y Thor 2 para un videojuego que estoy haciendo :)

**Comment by adrigm** [comment]
Sí, eso sería siguiendo los patrones establecidos. Si te animas crea el issue y asígnatelo. Cualquier duda, pregunta.

**Comment by RicardoRB** [comment]
Gracias, soy nuevo en esto de colaborar en un proyecto open source, espero ayudar en lo que pueda y aprender mucho.
